addons/parley/models/dialogue_sequence_ast.gd

```diff
@@ -357,10 +362,17 @@ func next(ctx: ParleyContext, current_node: ParleyNodeAst = null, dry_run: bool
 			continue
 		var next_node: ParleyNodeAst = filtered_next_nodes.front()
 		var next_type: Type = next_node.type
+		var rich_text_render_enabled: bool = Settings.get_setting(Constants.DIALOGUE_RICH_TEXT_RENDER)
 		match next_type:
 			Type.DIALOGUE:
+				if rich_text_render_enabled:
+					var text = ParleyUtils.richtext_renderer.render(next_node.get('text'))
+					next_node.set('text', text)
```

Comment:
Just FYI, we may need to coordinate with this PR here: https://github.com/bisterix-studio/parley/pull/28

I was thinking about it earlier and reading around for best practices and I think for the most flexible and easy way of defining translations, the function calls should probably run after the translations are run. wdyt?